addons/max-pod/max-pod.addon

```diff
@@ -0,0 +1,15 @@
+# Name: max-pod
+# Description: Add-on to increase the number of pods/node by passing the numnber of pods
+# Required-Vars: MAX_PODS  
+# OpenShift-Version: >3.6.0
+# Var-Defaults: MAX_PODS=20
+
+
+ssh docker exec -t origin /usr/bin/openshift ex config patch /var/lib/origin/openshift.local.config/node-localhost/node-config.yaml '--patch={"kubeletArguments": { "max-pods": ["#{MAX_PODS}"] } }' > tmpconfig
+
+ssh cat tmpconfig | sed "s/'/\'/" | sed "s/.$//" | sudo tee /var/lib/minishift/openshift.local.config/node-localhost/node-config.yaml > /dev/null
+ssh rm tmpconfig
+
```

Comment:
@anjannath you can directly use `docker exec` command like https://github.com/minishift/minishift-addons/blob/master/add-ons/cors/cors.addon.

Reply:
Also I think you can remove the intermediate file and directly apply the sed.

Reply:
Thanks @LalatenduMohanty i will try with that

Reply:
@LalatenduMohanty we need to use `ssh` as the patching process interects both with the container and the host vm.